transformer_engine/pytorch/module/linear.py

```diff
@@ -545,8 +553,8 @@ def backward(ctx, grad_output: torch.Tensor) -> Tuple[Union[torch.Tensor, None],
                                 if isinstance(inputmat_t_total, Float8Tensor)
                                 else inputmat_t_total
                             ),
-                            fwd_scale_inverses,
-                            tex.FP8FwdTensors.GEMM1_INPUT,
+                            inputmat_fp8_scale_inv,
+                            0,
```

Comment:
Why?

Reply:
`fwd_scale_inverses` is created with a clone operation (~20 us), while `input_fp8_scale_inv` is created with the scalar reciprocal kernel (~10 us).